<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TKNKasaPencere</class>
 <widget class="QDialog" name="TKNKasaPencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Marka</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Model</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_2</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Kasa Yapısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_3</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Anakart Yapısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_4</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Kasa Boyutu</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_5</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Ignored</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Güç Değeri</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_6</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Dahili Fan Sayısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_7</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Fan Yuva Sayısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_8</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Ekran Kartı Desteği</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_9</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>İşlemci soğutucu Yüksekliği</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_10</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>3.5&quot; Yuva Sayısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_11</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>2.5&quot; Yuva Sayısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_12</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>USB Sayısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_13</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>13</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Güç Kaynağı</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Var</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Yok</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_13"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>İptal Et</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TKNKasaPencere</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TKNKasaPencere</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_6</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
